producer.go

```diff
@@ -32,35 +29,30 @@ func (*Config) Name() string {
 
 // Component is the top level producer component.
 type Component struct {
-	Benthos *BenthosComponent
-	POST    *POSTComponent
-	Null    *NullComponent
+	POST *POSTComponent
+	Null *NullComponent
 }
 
 // NewComponent populates a ProducerComponent with defaults.
 func NewComponent() *Component {
 	return &Component{
-		Benthos: NewBenthosComponent(),
-		POST:    NewPOSTComponent(),
-		Null:    NewNullComponent(),
+		POST: NewPOSTComponent(),
+		Null: NewNullComponent(),
 	}
 }
 
 // Settings generates a default configuration.
 func (c *Component) Settings() *Config {
 	return &Config{
-		Type:    "BENTHOS",
-		Benthos: c.Benthos.Settings(),
-		POST:    c.POST.Settings(),
-		Null:    c.Null.Settings(),
+		Type: "BENTHOS",
```

Comment:
Following on with the comment above, I imagine our default Type should also be POST or Null